### ISSUE TITLE
BR not processed correctly

### ISSUE THREAD

**Issue by PrzemyslawKlys** [mysticmind/reversemarkdown-net#82]
It seems `<BR>` is left when processing this file. 

[PageProjectDescription.zip](https://github.com/mysticmind/reversemarkdown-net/files/5098879/PageProjectDescription.zip)

![image](https://user-images.githubusercontent.com/15063294/90682341-17f92680-e265-11ea-919e-784a67624ef6.png)

**Comment by SimonCropp** [comment]
so this `<body>\r\nb<br></body>` is converted to `<br>b  \r\n`

but 

`<div>\r\nb<br></div>` is converted to `\r\nb  \r\n`

so is leading newline after a body meant to be converted to a `<br/>`?

**Comment by SimonCropp** [comment]
so it seems to be done by:

```
   private static string ReplaceNewlineChars(string parentNodeName, string content)
        {
            var replaceStr = "<br>";

            if (parentNodeName == "p" || parentNodeName == "#document")
            {
                replaceStr = " ";
            }

            content = content.Replace("\r\n", replaceStr);
            content = content.Replace("\n", replaceStr);

            return content;
        }
```

shouldnt leading `\r\n` in a tag be replaced with `\r\n\r\n`?

**Comment by mysticmind** [comment]
Fixed via 4d6210fbe16501b08ece2cea9db3cf36f6e248cd

**Comment by mysticmind** [comment]
@SimonCropp Thanks for looking into this issue, I have applied a fix.
